src/main/java/com/flightstats/hub/spoke/SpokeReadContentDao.java

```diff
@@ -21,7 +24,7 @@
     private final static Logger logger = LoggerFactory.getLogger(SpokeReadContentDao.class);
 
     @Inject
-    private RemoteSpokeStore spokeStore;
+    private LocalReadSpoke localReadSpoke;
```

Comment:
...That class name was not misleading it all. 🤣 All of this is so much more understandable.